en/components/bulletgraph.md

```diff
@@ -0,0 +1,109 @@
+
+**Reserved area** (1) – This area spreads:
+
+- Along the scale – the Reserved area begins either at the edge of the control (when no Title area exists) or at the edge of the Title area (as in the pictures above) and ends at the edge of the control.
+
+- Across the scale –
+In horizontal orientation: the Reserved area begins at the bottom edge of the control and spreads upward as much as the height of the numbering labels is (depending on the size and the other formatting of the font used in the labels)
+In vertical orientation: the Reserved area begins at the left edge of the control and spreads to the right as much as the width of the numbering labels is (depending on how large the numbers of the measurements of the scale are and the size and the other formatting of the font used in the labels).
+
+**Graph area** (2) – The area for displaying the performance bar, the tick marks, the ranges, and, optionally, the numbering labels of the bullet graph. All extent-related properties for these visual elements (except for labels) are measured against its edges. The Graph area serves not as a placeholder but as a frame of reference for positioning the scale inside the control (More precisely, for positioning the visual elements of the scale).
+
+### Configurable Visual Elements Summary
+ ![](../images/Bullet_Graph_2.png)
+
+**Performance bar** (3) – This is the primary measure displayed by the control and is visualized as a bar.
+
+**Comparative marker** (4) – A measure which the performance bar measure compares against. It is visualized as a marker that runs perpendicular to the orientation of the performance bar.
+
+**Comparative range(s)** (5) – The ranges are visual elements that highlight a specified range of values on a scale. Their purpose is to visually communicate the qualitative state of the performance bar measure, illustrating at the same times the degree to which it resides within that state.
+
+**Tick marks** (6) –The tick marks serve as a visual division of the scale into intervals in order to increase the readability of the bullet graph.
+- Major tick marks – The major tick marks are used as primary delimiters on the scale. The frequency they appear at, their extents and style can be controlled by setting the corresponding properties.
+- Minor tick marks – The minor tick marks represent helper tick marks, which might be used to additionally improve the readability of the scale and can be customized in a way similar to the major ones.
+
+**Scale labels** (7) – The labels indicate the measures on the scale.
+
+**Border** (8) – The line visually delimiting the dimensions of the control.
+
+**Background** (9) – The background on which visual elements is placed is configurable in terms of pattern and color.
+ 
+**Tooltips** – show the values used to create the performance bar, the target value or the values, corresponding to the different ranges respectively.
```

Comment:
can you comment out tooltips for now? Tooltips are not working for any of gauges.